core/src/test/java/org/apache/jsieve/HeaderTest.java

```diff
@@ -57,6 +57,21 @@ public void testHeaderIsTrue() {
     /**
      * Test for Test 'header'
      */
+    @Test(expected = ThrowTestException.TestException.class)
+    public void testFoldedEncodedHeader() throws Exception {
```

Comment:
Why don't you use expected rule ? I think the name of the test does not rely speak ? Why is it throwing an exception ?